types/src/lib.rs

```diff
@@ -2618,6 +2618,28 @@ impl ModuleId {
         self.get(db).symbols.contains_key(name)
     }
 
+    pub fn import_symbol(self, db: &Database, name: &str) -> Option<Symbol> {
+        let Some(symbol) = self.symbol(db, name) else {
+            return None;
+        };
+
```

Comment:
`let else` isn't really needed here, so I replaced it with `self.symbol(...)?`, which is a tad shorter.